ros2_control_demo_robot/description/rrbot/rrbot_system_with_sensor.ros2_control.xacro

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="rrbot_system_with_sensor" params="name prefix">
+
+    <ros2_control name="${name}" type="system">
+      <hardware>
+        <plugin>ros2_control_demo_hardware/RRBotSystemWithSensorHardware</plugin>
+        <param name="example_param_write_for_sec">2</param>
+        <param name="example_param_read_for_sec">2</param>
```

Comment:
Propose to use the same parameters as in Example 1. This way, you don't mess with the control loop and provide an adjustment for the "slowdown"-effect for users to play with :)

Additionally, I propose a parameter on how much sensor measurements should change. So the idea would be to make a random change of the sensor's measurement +- around the parameter's valuer. What do you think about that?

```suggestion
        <param name="example_param_hw_start_duration_sec">2.0</param>
      <param name="example_param_hw_stop_duration_sec">3.0</param>
      <param name="example_param_hw_slowdown">2.0</param>
      <param name="example_param_max_sensor_change">5.0</param>
```

Reply:
If you agree with this change. Can you make PR in the ros-controls/roadmap repo to the example description?

Reply:
> Propose to use the same parameters as in Example 1. This way, you don't mess with the control loop and provide an adjustment for the "slowdown"-effect for users to play with :)
> 
> Additionally, I propose a parameter on how much sensor measurements should change. So the idea would be to make a random change of the sensor's measurement +- around the parameter's valuer. What do you think about that?

I understand your thought about the 'start_sec' and 'stop_sec' here. I added the changes accordingly. But I didn't get the usage of 'slowdown' and 'sensor_change' here. Can you please elaborate with an example please?



> If you agree with this change. Can you make PR in the ros-controls/roadmap repo to the example description?

I didn't understand exactly. Are you saying to update "https://github.com/ros-controls/roadmap/blob/master/design_drafts/ros2_control_documentation.rst" once we finalize the robot description?

Reply:
We should update [this](https://github.com/ros-controls/roadmap/blob/master/design_drafts/components_architecture_and_urdf_examples.md) file.

Regarding `slowdown` please check the implementation of Example 1.

Regarding `sensor_change` I imagine the following (in pseudo code):
```
sensor_value = random between 0 and sensor_change value
```
This for each sensor state.

If you have further questions do not hesitate.

Reply:
[This](https://github.com/ros-controls/roadmap/blob/master/design_drafts/components_architecture_and_urdf_examples.md) will be updated in separate PR?

Reply:
Added the PR https://github.com/ros-controls/roadmap/pull/40